crates/re_space_view_spatial/src/contexts/mod.rs

```diff
@@ -23,6 +23,7 @@ use re_viewer_context::{
 /// Context objects for a single entity in a spatial scene.
 pub struct SpatialSceneEntityContext<'a> {
     pub world_from_entity: glam::Affine3A,
+    pub radii_scale_factor: Option<f32>,
```

Comment:
This needs a docstring: what is it? And what is the difference between `None` and `Some(1.0)`?